src/network/networkd-dhcp6.c

```diff
@@ -20,13 +20,19 @@
 
 #include <netinet/ether.h>
 #include <linux/if.h>
+#include "sd-radv.h"
 
 #include "sd-dhcp6-client.h"
 
+#include "hashmap.h"
 #include "hostname-util.h"
 #include "network-internal.h"
 #include "networkd-link.h"
 #include "networkd-manager.h"
+#include "siphash24.h"
+#include "radv-internal.h"
+
+static Hashmap *dhcp6_prefixes = NULL;
```

Comment:
Hmmm, a static variable, that doesn't look right. I'm pretty sure that this should be located somewhere underneath the manager object.

Reply:
Added new patch for this, and placed hashmap in Manager.